javase/pom.xml

```diff
@@ -31,6 +31,11 @@
       <artifactId>jcommander</artifactId>
       <version>1.48</version>
     </dependency>
+    <dependency>
+      <groupId>net.imagej</groupId>
+      <artifactId>ij</artifactId>
+      <version>1.48r</version>
```

Comment:
OK this is still the older version but I can fix that later. Is there a test image that isn't read by ImageIO? I probably need a test that shows this works.